### DISCUSSION TITLE
How to recover used Unknown Entities

### DISCUSSION THREAD

**Discussion by Percherie** [frenck/spook#802] [Q&A]
Hello,

When Spook indicates unknown entities in use, I'd like to be able to retrieve their name to process them with an automaton... for example, restart HA so that they are correctly taken into account.

I've tried with {{ states.event.repair }} but the information available doesn't match the list of entities displayed in Spook.

How can I retrieve this information?

**Comment by frenck** [comment]
No, that information cannot be made available as separate meta data from an issue.

**Comment by Percherie** [comment]
In 99% of cases, Spook sends me anomalies for the same integration not loading correctly. 
Checking both the entity and Spook seems redundant but at least I'm sure I'm targeting the problem. 

If the condition is true, I declare a notification, wait 1 min (to avoid boot loops) and restart HA.

It's not great but it works

`{{ states('sensor.bbox_uptime') == 'unknown' and states('sensor.active_issues') > '0' }}`

**Comment by frenck** [comment]
> Spook sends me anomalies for the same integration not loading correctly.

That should not be possible, meaning the integration doesn't register its entities in the entity registry. It should.

../Frenck
